content/install/server/database.es.md

```diff
@@ -0,0 +1,58 @@
++++
+date = "2017-04-15T14:39:04+02:00"
+title = "Configuraciones de base de datos"
+url = "database-settings"
+
+[menu.install]
+  identifier = "database-settings"
+  parent = "install_server"
+  weight = 1
++++
+
+Esta guía provee instrucciones para usar motores de almacenaje alternativos. Por favor nota que esto es opcional. El motor de almacenaje por defecto es una base de datos SQLite embebida que no requiere instalarse o configurarse.
```

Comment:
I would be more direct, since we're talking about databases, I would say "motores de bases de datos"... This because a "motor de almacenaje" could be the subtle difference between MyISAM and InnoDB in MySQL, which is not what the topic talks about.